tests/test_envs.py

```diff
@@ -163,3 +164,67 @@ def test_pickle_env(env_spec):
     data_equivalence(env.step(action), pickled_env.step(action))
     env.close()
     pickled_env.close()
+
+
+_test_robot_env_reset_list = ["Fetch", "HandReach"]
+
+
+@pytest.mark.parametrize(
+    "spec",
+    [
+        spec
+        for spec in non_mujoco_py_env_specs
+        if np.any([tar in spec.id for tar in _test_robot_env_reset_list])
+    ],
+    ids=[
+        spec.id
+        for spec in non_mujoco_py_env_specs
+        if np.any([tar in spec.id for tar in _test_robot_env_reset_list])
+    ],
+)
+def test_robot_env_reset(spec):
+    """Check initial state of robotic environment, i.e. Fetch and Shadow Dexterous Hand Reach,
+    whether their initial states match the description in the documentation."""
+
+    def _test_initial_states(env, seed=None):
+        diag_dict = {}
+
+        init_obs = env.reset(seed=seed)
```

Comment:
This should not be a tuple, instead split to
`init_obs, init_info = env.reset(seed=seed)`
and make the changed below accordingly

for improved readability

Reply:
I'm deleting it. I just copied the code from notebook for I used for inspection.